scene/gui/line_edit.cpp

```diff
@@ -818,6 +818,7 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 	if ((!k->is_pressed() && alt_start && k->get_keycode() == Key::ALT) || (alt_start_no_hold && (k->is_action("ui_text_submit", true) || k->is_action("ui_accept", true)))) {
 		alt_start = false;
 		alt_start_no_hold = false;
+		int prev_len = false;
```

Comment:
```suggestion
		int prev_len = text.length();
```
Copy paste error I assume